### PR TITLE
Remove the enterprise contract task

### PR DESCRIPTION
**Pull request by joejstuart**
The EC task has moved to https://github.com/hacbs-contract/ec-cli, so this PR removes the task from this repo and updates the EC integration pipeline to use the new bundle location.

### CODE REVIEW

**Comment by zregvart** [comment]
@redhat-appstudio/integration-team please take note of this change

File: pipelines/enterprise-contract.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ spec:
         paths can be provided by using the ":" separator.
       default: ""
   results:
-    - name: REPORT
-      value: "$(tasks.verify.results.REPORT)"
+    - name: HACBS_TEST_OUTPUT
+      value: "$(tasks.verify.results.HACBS_TEST_OUTPUT)"
```
Comment (simonbaird) [inline pipelines/enterprise-contract.yaml:43]: Might be worth mentioning the reason why this change was needed.

**Review by simonbaird — APPROVED** [review]
